live-build/misc/ansible-roles/appliance-build.install-delphix-s3-packages/tasks/main.yml

```diff
@@ -56,10 +57,15 @@
   args:
     chdir: "{{ tempdir.path }}"
 
-- apt:
-    deb: "{{ item }}"
-  with_fileglob:
-    - "{{ tempdir.path }}/*.deb"
+#
+# We use apt-get here instead of ansible's apt module to install all the debs
+# at the same time, so that any dependencies between provided packages are
+# resolved. We use 'shell' instead of 'command' to resolve the glob (*.deb).
+#
+- name: Install packages
+  shell: "apt-get install -y ./*.deb"
```

Comment:
We might also want to set the environment variable: `DEBIAN_FRONTEND=noninteractive ` .. and use the apt-get options: `-o DPkg::Options::=--force-confold -o DPkg::Options::=--force-confdef`